controllers/home.js

```diff
@@ -26,7 +26,9 @@ function home(req, res) {
 
 function notFound(req, res) {
   res.status(404);
-  res.render("error", { message: req.fluentFormat("home-not-found") });
+  res.render("subpage", {
+    headline: "Error",
```

Comment:
I ses this was a bug in the old `error.hbs` template too, but this needs to be wrapped in `req.fluentFormat("error")` ?